core/src/integration-test/java/com/scalar/db/storage/jdbc/ConsensusCommitAdminIntegrationTestWithJdbcDatabase.java

```diff
@@ -1,13 +1,49 @@
 package com.scalar.db.storage.jdbc;
 
+import com.scalar.db.exception.storage.ExecutionException;
 import com.scalar.db.transaction.consensuscommit.ConsensusCommitAdminIntegrationTestBase;
 import java.util.Properties;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledIf;
 
 public class ConsensusCommitAdminIntegrationTestWithJdbcDatabase
     extends ConsensusCommitAdminIntegrationTestBase {
 
+  private static final String PROP_JDBC_URL = "scalardb.jdbc.url";
+  private static final String DEFAULT_JDBC_URL = "jdbc:mysql://localhost:3306/";
+
   @Override
   protected Properties getProps(String testName) {
     return JdbcEnv.getProperties(testName);
   }
+
+  // Since SQLite doesn't have persistent namespaces, some behaviors around the namespace are
+  // different from the other adapters. So disable several tests that check such behaviors.
+
+  private boolean isSqlite() {
+    String jdbcUrl = System.getProperty(PROP_JDBC_URL, DEFAULT_JDBC_URL);
+    return jdbcUrl.startsWith("jdbc:sqlite:");
```

Comment:
Sorry, I missed this in the previous PR for some reason. Can you please add a method doing this in the `JdbcEnv` class, and call the new method here?

Reply:
68ea3f1d30d63666c92929f62cb225869372becb